pmlib/Resonator2D.py

```diff
@@ -0,0 +1,77 @@
+
+from math import sqrt
+
+import numpy as np
+from scipy.sparse import dia_matrix,identity
+
+from .ResonatorBase import ResonatorBase
+from .sparse_diff_matr_2d import (
+    BOUNDARY_CONDITIONS_2D, laplacian_matrix_2d, biharmonic_matrix_2d)
+
+
+class Resonator2D(ResonatorBase):
+    valid_boundaries = BOUNDARY_CONDITIONS_2D
+
+    def __init__(self, gamma=200, kappa=1, b1=0, b2=0,
+                 boundary='SSSS', epsilon=1):
+        super().__init__(gamma, kappa, b1, b2, boundary)
+        self._epsilon = epsilon
+        # self.h = None
```

Comment:
I think these commented out lines can be deleted?

Reply:
Yes, they were there just to mark that those are attributes created later.